device/mellanox/x86_64-nvidia_sn5600-r0/Mellanox-SN5600-C256/port_config.ini

```diff
@@ -0,0 +1,275 @@
+Ethernet451    451                                    etp57d    57       100000   rs    off     4
+Ethernet452    452                                    etp57e    57       100000   rs    off     5
+Ethernet453    453                                    etp57f    57       100000   rs    off     6
+Ethernet454    454                                    etp57g    57       100000   rs    off     7
+Ethernet455    455                                    etp57h    57       100000   rs    off     8
+Ethernet464    464                                    etp59a    59       100000   rs    off     1
+Ethernet465    465                                    etp59b    59       100000   rs    off     2
+Ethernet466    466                                    etp59c    59       100000   rs    off     3
+Ethernet467    467                                    etp59d    59       100000   rs    off     4
+Ethernet468    468                                    etp59e    59       100000   rs    off     5
+Ethernet469    469                                    etp59f    59       100000   rs    off     6
+Ethernet470    470                                    etp59g    59       100000   rs    off     7
+Ethernet471    471                                    etp59h    59       100000   rs    off     8
+Ethernet480    480                                    etp61a    61       100000   rs    off     1
+Ethernet481    481                                    etp61b    61       100000   rs    off     2
+Ethernet482    482                                    etp61c    61       100000   rs    off     3
+Ethernet483    483                                    etp61d    61       100000   rs    off     4
+Ethernet484    484                                    etp61e    61       100000   rs    off     5
+Ethernet485    485                                    etp61f    61       100000   rs    off     6
+Ethernet486    486                                    etp61g    61       100000   rs    off     7
+Ethernet487    487                                    etp61h    61       100000   rs    off     8
+Ethernet496    496                                    etp63a    63       100000   rs    off     1
+Ethernet497    497                                    etp63b    63       100000   rs    off     2
+Ethernet498    498                                    etp63c    63       100000   rs    off     3
+Ethernet499    499                                    etp63d    63       100000   rs    off     4
+Ethernet500    500                                    etp63e    63       100000   rs    off     5
+Ethernet501    501                                    etp63f    63       100000   rs    off     6
+Ethernet502    502                                    etp63g    63       100000   rs    off     7
+Ethernet503    503                                    etp63h    63       100000   rs    off     8
+Ethernet512    512                                    etp65     65        25000   rs
```

Comment:
Since this one contains 1 25G port, do you mind to rename the SKU to C256X1?

Reply:
I agree, but as far as I am aware this should be C256A1 (e.g. [Mellanox-SN4700-A96C8V8](https://github.com/sonic-net/sonic-buildimage/tree/master/device/mellanox/x86_64-mlnx_msn4700-r0/Mellanox-SN4700-A96C8V8) )

Reply:
hi @DavidZagury , not really, the last port is 25Gb, and it should be X not A. @yxieca can correct me, if I am wrong.

Reply:
@r12f My notes says that the speed / name  map is:
10Gb  - "S"
25Gb  - "A"
50Gb  - "D"
100Gb - "C"
200Gb - "V"
400Gb - "O"

And this matches what we see in other SKUs such as SN4700-A96C8V8.
But I don't know where to find an official answer to that.

@liat-grozovik